app/share.tsx

```diff
@@ -8,7 +8,7 @@ import { Provider } from 'react-redux';
 import { defaultTheme, newThemeState, subscribeTheme, unsubscribeTheme } from './utils/theme';
 import UserPreferences from './lib/userPreferences';
 import Navigation from './lib/ShareNavigation';
-import store from './lib/createStore';
+import { store } from './lib/auxStore';
```

Comment:
It's not working this way.
Share extension is another app, so we'll need to initialize it the same way we do on `app/index.tsx`